tests/fixtures/edx/navigational.py

```diff
@@ -0,0 +1,95 @@
+"""Navigational event factory definitions"""
+
+from factory import SubFactory
+from faker import Faker
+
+from ralph.models.edx.navigational import (
+    NavigationalBrowserEventEventFieldModel,
+    PageCloseBrowserEventModel,
+    SeqGotoBrowserEventModel,
+    SeqNextBrowserEventModel,
+    SeqPrevBrowserEventModel,
+)
+
+from .base import BaseEventFieldFactory
+from .browser import BaseBrowserEventFactory
+
+Faker.seed(1)
+FAKE = Faker()
```

Comment:
```suggestion
from .base import FAKE, BaseEventFieldFactory
from .browser import BaseBrowserEventFactory
```
FAKE is already created in `.base`, we could reuse it ?

Reply:
You're right, thanks! I did not think of reusing it